python/cudf/cudf/testing/fuzzer.py

```diff
@@ -0,0 +1,92 @@
+import datetime
+import functools
+import json
+import logging
+import os
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+try:
+    lru_cache = functools.lru_cache
+except Exception:
+    import functools32
+
+    lru_cache = functools32.lru_cache
```

Comment:
Why do we need to try / except this? When does `functools.lru_cache` not exist?

Reply:
This was a copy-paste. Didn't know why flake8 doesn't complain for this..